docs/resource-duration.md

```diff
@@ -39,24 +39,23 @@ set `requests` and/or `limits` for all containers.
 
 ### Example
 
-A pod that runs for 3min, with a CPU limit of `2000m`, no memory request and an `nvidia.com/gpu`
+A pod that runs for 3min, with a CPU limit of `2000m`, a memory limit of `1Gi` and an `nvidia.com/gpu`
 resource limit of `1`:
 
 ```text
 CPU:    3min * 2000m / 1000m = 6min * (1 cpu)
-Memory: 3min * 100Mi / 1Gi   = 18sec * (100Mi memory)
-GPU:    3min * 1     / 1     = 2min * (1 nvidia.com/gpu)
+Memory: 3min * 1Gi / 100Mi   = 30min * (100Mi memory)
+GPU:    3min * 1     / 1     = 3min * (1 nvidia.com/gpu)
 ```
 
 ### Web/CLI reporting
 
 Both the web and CLI give abbreviated usage, like `9m10s*cpu,6s*memory,2m31s*nvidia.com/gpu`. In
 this context, resources like `memory` refer to the "base amounts".
 
-For example, `memory` means "amount of time a resource requested 1Gi of memory." If a container only
-uses 100Mi, each second it runs will only count as a tenth-second of `memory`.
+For example, `memory` means "amount of time a resource requested `100Mi` of memory." If a container only
+uses `10Mi`, each second it runs will only count as a tenth-second of `memory`.
 
 ## Rounding Down
 
-For short running pods (<10s), the memory value may be 0s. This is because the default is `100Mi`,
-but the denominator is `1Gi`.
+For a short running pods (<10s), if the memory requests of this pod is also small (for example, `10Mi`), then the memory value may be 0s. This is because the dominator is `100Mi`.
```

Comment:
```suggestion
For a short running pods (<10s), if the memory request is also small (for example, `10Mi`), then the memory value may be 0s. This is because the denominator is `100Mi`.
```

can shorten this a bit. also typo on denominator